app/build.gradle.kts

```diff
@@ -153,6 +153,7 @@ dependencies {
     implementation(libs.androidx.core.splashscreen)
     implementation(libs.recaptcha)
     implementation(libs.workmanager)
+    implementation(libs.splitties.toast)
```

Comment:
You can use `core2.splitties.toast` instead.

Reply:
That's right, thanks for noticing!